benchmark/operations/test/test_create_dataset.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Create Dataset operation tests for the control service benchmarks.
+"""
+from uuid import uuid4
+
+from ipaddr import IPAddress
+from zope.interface.verify import verifyClass
+
+from twisted.internet.task import Clock
+from twisted.trial.unittest import SynchronousTestCase
+
+from flocker.apiclient import FakeFlockerClient, Node
+
+from benchmark.cluster import BenchmarkCluster
+from benchmark._interfaces import IOperation, IProbe
+from benchmark.operations.create_dataset import (
+    CreateDatasetConvergence, CreateDatasetConvergenceProbe, EmptyClusterError,
+)
+
+
+class CreateDatasetTests(SynchronousTestCase):
+    """
+    CreateDataset operation tests.
+    """
+
+    def test_implements_IOperation(self):
+        """
+        CreateDatasetConvergence provides the IOperation interface.
+        """
+        verifyClass(IOperation, CreateDatasetConvergence)
```

Comment:
Not necessarily relevant in this case, but it's more useful to do `verifyObject` on an instance, cause sometimes the class can't do everything and so `verifyObject` on an instance does more validation. I believe e.g. `Attribute` is only verified by `verifyObject`.